### PR TITLE
Feature: Document undocumented config vars

### PR DESCRIPTION
**Pull request by bohme**
Asana-task: [Required review/test app env vars are not documented in the README](https://app.asana.com/0/1142794766483633/1189653570202377/f)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,8 +71,11 @@ your own repository.
     - Staging: `<PROJECT-NAME>-staging`
     - Production: `<PROJECT-NAME>-production`
 3. Turn on "Review Apps" from the Pipeline's page
-4. Set `MAIL_FROM` in your Heroku config vars (review, staging and production).
-5. Set `DOMAIN_NAME` in your Heroku config vars (staging and production), links in email will point to the provided domain name.
+4. Set Heroku config vars:
+   - `SEED_ADMIN_EMAIL` (review, staging and production)
```
Comment (prrrnd) [inline README.md:75]: production and seeds?

Comment (bohme) [reply README.md:75]: uhhh you are right 🤔

Comment (jeppester) [reply README.md]: Currently the master branch runs the seeds in the production env, so this content is correct.
It is however going to be fixed shortly :-)

**Review by hcarreras — APPROVED** [review]
👍

**Pull request merged**